### DISCUSSION TITLE
使用bing翻译频繁失效~

### DISCUSSION THREAD

**Discussion by charon979** [chunibyocola/sc-translator-crx#161] [Q&A] [answered]
<img width="471" alt="截屏2023-12-11 20 40 50" src="https://github.com/chunibyocola/sc-translator-crx/assets/116360738/296a01df-61e1-4e7e-bae2-72645fdd5331">

**Comment by chunibyocola** [comment, marked as answer]
这是 Bing 那边启用了一些限制，这个限制有时会无缘无故地出现，且会传染到官方网站那边（也可能是相反的），自从一次更新之后就有这种现象。目前可以在触发这种状况之后前往 [cn.bing.com/translator](https://cn.bing.com/translator)那边去验证一下就可以继续使用。

我试试在新版本中更新一下接口看看会不会有所改善。

**Reply by charon979** [reply]
好嘟！十分感谢
